vocode/streaming/synthesizer/play_ht_synthesizer.py

```diff
@@ -78,14 +82,26 @@ async def create_speech_uncached(
                 timeout=ClientTimeout(total=15),
             )
             if not response.ok:
-                raise Exception(f"Play.ht API error status code {response.status}")
+                raise PlayHTV1APIError(
+                    f"Play.ht API error status code {response.status}",
+                    (await response.json()).get(
+                        "error_message",
+                        f"Request to Play.HT failed with status code {str(response.status)}",
+                    ),
+                )
             if response.status == 429 and attempt < max_backoff_retries - 1:
                 await asyncio.sleep(backoff_retry_delay)
                 backoff_retry_delay *= 2  # Exponentially increase delay
                 continue
 
             if not response.ok:
-                raise Exception(f"Play.ht API error status code {response.status}")
+                raise PlayHTV1APIError(
+                    f"Play.ht API error status code {response.status}",
+                    (await response.json()).get(
```

Comment:
Do we need another error handler for this step where we're getting the JSON from the response? I think typically it's best to avoid things that can fail in an exception handler

Reply:
I don't recall a scenario where the json will fail to load but can add one in to be extra safe. Would you prefer another try except? Or some other structure in the exception handling?

Reply:
It can fail if the json isn't properly formatted, which we don't have control over if it's coming from a provider. This error assumes that we still get a valid JSON in the response. If that's the case, maybe store the response JSON in a variable and use it here -- defaulting to some generic message if the JSON is empty/null-ish?